tools/ci_build/github/linux/docker/scripts/requirements.txt

```diff
@@ -1,7 +1,5 @@
 cerberus
-numpy==1.21.6 ; python_version < '3.11'
-numpy==1.24.2 ; python_version == '3.11'
-numpy==1.26.0 ; python_version >= '3.12'
+numpy
```

Comment:
Regarding openvino CI failure:
Perhaps this should be `numpy >= 1.20` because we're also installing `onnx==1.16.1` below. Similarly, any other requirements.txt files that use onnx 1.16.1 should probably also specify numpy >= 1.20

https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1415326&view=logs&j=8c952c3b-dbd4-5685-50d1-7e2a31304cbf&t=ab7852eb-c27a-552f-5934-55aea3e695cb&l=4511
```
#9 205.3 ERROR: onnx 1.16.1 has requirement numpy>=1.20, but you'll have numpy 1.17.4 which is incompatible. 
#9 206.5 Attempting uninstall: setuptools 
#9 206.5 Found existing installation: setuptools 45.2.0 
#9 206.5 Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr 
#9 206.5 Can't uninstall 'setuptools'. No files were found to uninstall. 
#9 206.9 Attempting uninstall: wheel 
#9 206.9 Found existing installation: wheel 0.34.2 
#9 206.9 Not uninstalling wheel at /usr/lib/python3/dist-packages, outside environment /usr 
#9 206.9 Can't uninstall 'wheel'. No files were found to uninstall. 
```